src/parser.ts

```diff
@@ -57,20 +57,43 @@ export function exportSchedule(sheet: XLSX.WorkSheet) {
         .filter(row => meetingPatternsRegex.test(row['Meeting Patterns'] ?? ''))
         .filter(row => /^(Registered)|(Waitlisted)/.test(row['Registration Status'] ?? ''))
         .forEach(row => {
-            const [days, startTime, endTime, location] =
+            const [days, startTime, endTime, spot] =
                 meetingPatternsRegex.exec(row['Meeting Patterns'] ?? '')!.slice(1)
-            schedule.addEvent(
-                row['Course Listing']!,
-                `${row['Section']} with ${row['Instructor']}`,
-                location,
-                `${row['Start Date']} ${startTime}`,
-                `${row['Start Date']} ${endTime}`,
-                {
-                    freq: 'WEEKLY',
-                    until: `${row['End Date']} ${endTime}`,
-                    byday: convertDayOfWeekFormat(days),
-                },
-            )
+            const [startHour, startMinute] =
+                startTime.split(":")
+            const [endHour, endMinute] =
+                endTime.split(":")
+            const [month, day, badYear] =
+                row['Start Date']!.split("/")
+            const [endMonth, endDay, badEndYear] =
+                row['End Date']!.split("/")
+            // Workday for some ungodly reason puts the year as two digits.
```

Comment:
Workday is so so screwy; so this frankly doesn't surprise me.

For instance, I remember having to hit an API endpoint that ends something like `workday.com/foo/bar.xlsx` that is called from the XLSX download button, but returns *JSON*, because that makes sense.

Reply:
I love Workday.

The workaround for this was super jank but I'm betting on Workday being changed out before 2100.